composer.json

```diff
@@ -1,9 +1,15 @@
 {
     "name": "statamic/starter-kit-podcaster",
+    "license": "proprietary",
     "extra": {
         "statamic": {
             "name": "Starter Kit Podcaster",
             "description": "Starter Kit Podcaster starter kit"
         }
+    },
+    "config": {
+        "allow-plugins": {
+            "pixelfear/composer-dist-plugin": true
```

Comment:
Netlify build fails without allowing this plugin.
With this line it fails a bit later :-D